open-craft/opencraft
# [BB-3869] Update Hero page

Updates hero page to conform to the new UI

**Related Tickets** [BB-3869](https://tasks.opencraft.com/browse/BB-3869) [Gitlab#721](https://gitlab.com/opencraft/dev/opencraft/-/issues/721)

**Testing Instructions**
1. Checkout to this branch
2. Visit http://localhost:3000/console/theming/hero
3. Verify that the UI looks correct and matches [this mock](https://app.zeplin.io/project/5ea850e04326be273e4bcf5a/screen/5ffea861fa26d61b53abb3b9)
3. Verify that all the components from old Hero page are moved to sidebar.
4. Verify that updating values in sidebar gets reflected in `BetaTestApplication`

**Screenshots**
Hero Edit screen
![Screenshot from 2021-04-09 18-53-33](https://user-images.githubusercontent.com/18226212/114187767-fd93ca80-9965-11eb-9712-7534d4e10c8a.png)
